### PR TITLE
[Intel MKL] Enable DepthwiseConv2D + BiasAdd (+ Relu/Relu6/Elu) fusion

### PR DESCRIPTION
**Pull request by CuiYifeng**

### CODE REVIEW

**Comment by CuiYifeng** [comment]
@penpornk Could you please review this PR?

**Comment by penpornk** [comment]
I'm sorry for the delay! I'll review this in a few days.

**Review by penpornk — CHANGES_REQUESTED** [review]
Thank you for the PR and I'm so sorry for the delay!
I've requested another review from @ezhulenev since this PR also modifies grappler.

**Comment by CuiYifeng** [comment]
@penpornk Thanks for the comments. I have updated my code, please check. Thank you!

File: tensorflow/core/graph/mkl_layout_pass_test.cc [hunk 1]
```diff
@@ -1921,6 +1789,56 @@ REGISTER_TEST_ALL_TYPES(NodeRewrite_FusedConv2D_Positive6);
...
+        "node { name: 'D' op: '_FusedDepthwiseConv2dNative'"                   \
+        " attr { key: 'T'                value { type: " #T " } }"             \
+        " attr { key: 'num_args'         value { i: 1 } }"                     \
+        " attr { key: 'data_format'      value { s: 'NCHW' } }"                \
+        " attr { key: 'strides'          value { list: {i: 1, i:1, i:1, i:1} " \
+        "} }"                                                                  \
+        " attr { key: 'padding'          value { s: 'SAME' } }"                \
+        " attr { key: 'dilations'        value { list: {i: 1, i:1, i:1, i:1} " \
+        "} }"                                                                  \
+        " attr { key: 'fused_ops'        value { list: " FUSED_OPS" } }"       \
```
Comment (penpornk) [inline tensorflow/core/graph/mkl_layout_pass_test.cc:1808]: Minor nit: Could we add a space after `FUSED_OPS`? Just for uniformity.
```suggestion
        " attr { key: 'fused_ops'        value { list: " FUSED_OPS " } }"       \
```

File: tensorflow/core/grappler/optimizers/mkl_remapper_test.cc [hunk 2]
```diff
@@ -234,6 +172,8 @@ TEST_F(MklRemapperTest, FuseDepthwiseConv2DWithBiasAndActivation) {
         return ops::Identity(fetch, ops::Relu6(activate, bias_add));
       } else if (activation == "Elu") {
         return ops::Identity(fetch, ops::Elu(activate, bias_add));
+      } else if (activation == "None") {
+        return ops::Identity(s.WithOpName("fetch"), bias_add);
```
Comment (penpornk) [inline tensorflow/core/grappler/optimizers/mkl_remapper_test.cc:176]: The `return` outside of if already does this. So I don't think we need this `else if`.

**Review by penpornk — CHANGES_REQUESTED** [review]
Thank you for all the fixes! I have two more minor nits. Apart from that, this PR looks good to me.

**Comment by CuiYifeng** [comment]
@penpornk Thank you for your comments! The two nits are fixed, please check.

**Review by penpornk — COMMENTED** [review]
Thank you for the fixes! This PR looks good to me. Not approving yet because the PR only needs one approval to be `ready to pull`.

@ezhulenev Could you please help take a look if the grappler parts are okay? Thank you!

**Review by ezhulenev — APPROVED** [review]
Thanks Penporn for the review, Grappler part looks good to me.

**Review by penpornk — APPROVED** [review]
Thank you for the quick review, Eugene!

**Pull request merged**